<?xml version="1.0" encoding="UTF-8"?>
<testsuites>
  <testsuite name="" tests="3" assertions="8" errors="0" warnings="0" failures="0" skipped="0" time="0.515756">
    <testsuite name="login" tests="3" assertions="8" errors="0" warnings="0" failures="0" skipped="0" time="0.515756">
      <testsuite name="Tests\Feature\Auth\LoginTest" file="/home/keshav/projects/user-directory/tests/Feature/Auth/LoginTest.php" tests="3" assertions="8" errors="0" warnings="0" failures="0" skipped="0" time="0.515756">
        <testcase name="testExample" class="Tests\Feature\Auth\LoginTest" classname="Tests.Feature.Auth.LoginTest" file="/home/keshav/projects/user-directory/tests/Feature/Auth/LoginTest.php" line="17" assertions="1" time="0.183777"/>
        <testcase name="test_user_can_view_a_login_form" class="Tests\Feature\Auth\LoginTest" classname="Tests.Feature.Auth.LoginTest" file="/home/keshav/projects/user-directory/tests/Feature/Auth/LoginTest.php" line="42" assertions="2" time="0.140917"/>
        <testcase name="user_can_login_with_correct_credentials" class="Tests\Feature\Auth\LoginTest" classname="Tests.Feature.Auth.LoginTest" file="/home/keshav/projects/user-directory/tests/Feature/Auth/LoginTest.php" line="52" assertions="5" time="0.191062"/>
      </testsuite>
    </testsuite>
  </testsuite>
</testsuites>
